native/swift/Sources/wordpress-api/SafeRequestExecutor.swift

```diff
@@ -158,6 +151,16 @@ final class WpRequestExecutor: SafeRequestExecutor {
         }
     }
     // swiftlint:enable force_cast
+
+    // swiftlint:disable force_try
+    func sleep(millis: UInt64) async {
+        if #available(macOS 13.0, iOS 16.0, tvOS 16.0, watchOS 9.0, *) {
+            try! await Task.sleep(for: .milliseconds(millis))
+        } else {
+            try! await Task.sleep(nanoseconds: millis * 1000)
+        }
```

Comment:
I think it's best to `try?` here. I believe `sleep` throws a `CancellationError`, when the task is cancelled while `sleep`ing.

Reply:
Yep that's right! The underlying protocol doesn't return a `Result` type so we can't make this function `throw`-ing, but it's on the list to fix this up for sure